homeassistant/components/slide_local/models.py

```diff
@@ -0,0 +1,20 @@
+"""Dataclass models for the slide_local integration."""
+
+from dataclasses import dataclass
+
+from goslideapi import GoSlideLocal as SlideLocalApi
+
+from homeassistant.config_entries import ConfigEntry
+
+type SlideConfigEntry = ConfigEntry[SlideData]
+
+
+@dataclass
+class SlideData:
```

Comment:
Why do we store the rest of the data in here as well? At all the places we have access to entry.runtime_data we also have access to entry.data, so we don't have to store this twice imo